core-plugins/src/test/java/co/cask/hydrator/plugin/batch/ETLBatchTestBase.java

```diff
@@ -141,6 +141,28 @@ public static void setupTest() throws Exception {
                       SSHAction.class);
   }
 
+  protected boolean checkFileType(TimePartitionedFileSet fileSet, String expectedType) throws IOException {
```

Comment:
This is not generic and wouldn't be able to handle the time partition specification.